### PR TITLE
Clean up words in F# install page

### PR DESCRIPTION
**Pull request by cartermp**

### CODE REVIEW

File: docs/fsharp/get-started/install-fsharp.md [hunk 1]
```diff
@@ -28,30 +28,30 @@ You must have [git installed](https://git-scm.com/download) and available on you
 
 ### [macOS](#tab/macos)
 
-[Mono](http://www.mono-project.com) is used for [F# Interactive](https://docs.microsoft.com/dotnet/fsharp/tutorials/fsharp-interactive/) support. The easiest way to install Mono on macOS is via Homebrew. Simply type the following into your terminal:
+[Mono](http://www.mono-project.com) is used for [F# Interactive](../tutorials/fsharp-interactive.md) support. The easiest way to install Mono on macOS is via Homebrew. Simply type the following into your terminal:
```
Comment (rpetrusha) [inline docs/fsharp/get-started/install-fsharp.md]: The link is broken: ../tutorials/fsharp-interactive/index.md

File: docs/fsharp/get-started/install-fsharp.md [hunk 2]
```diff
@@ -28,30 +28,30 @@ You must have [git installed](https://git-scm.com/download) and available on you
...
 brew install mono
 ```
 
-You must also install the [.NET Core SDK](https://www.microsoft.com/net/download).
+Also install the [.NET Core SDK](https://www.microsoft.com/net/download).
 
 ### [Linux](#tab/linux)
 
-[Mono](https://www.mono-project.com) is used for [F# Interactive](https://docs.microsoft.com/dotnet/fsharp/tutorials/fsharp-interactive/) support. If you're on Debian or Ubuntu, you can use the following:
+[Mono](https://www.mono-project.com) is used for [F# Interactive](../tutorials/fsharp-interactive.md) support. If you're on Debian or Ubuntu, you can use the following:
```
Comment (rpetrusha) [inline docs/fsharp/get-started/install-fsharp.md]: The link is broken: ../tutorials/fsharp-interactive/index.md

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @cartermp, except that the two links to the interactive install topic are broken.

**Comment by mairaw** [comment]
My change should fix that.

**Comment by cartermp** [comment]
Thanks @mairaw, sorry for the mess on my end

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
This is ready to merge, @cartermp.

**Pull request merged**